org.eclipse.xtext.common.types.eclipse.tests/testdata/org/eclipse/xtext/common/types/testSetups/TestAnnotation.java

```diff
@@ -35,6 +35,8 @@
 	Class<? extends CharSequence> charSequenceClass();
 	Class<?>[] classArray();
 	
+	static final Object NOT_A_CONSTANT = new Object();
```

Comment:
no a constant means the right hand side?

Reply:
Yes. I should have named it 'STATIC_FIELD'